[2023-03-22 08:34:33,267] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: load_to_datamart.sukses_Proses_Load_Datamart manual__2023-03-22T08:34:25.490061+00:00 [queued]>
[2023-03-22 08:34:33,294] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: load_to_datamart.sukses_Proses_Load_Datamart manual__2023-03-22T08:34:25.490061+00:00 [queued]>
[2023-03-22 08:34:33,294] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-03-22 08:34:33,295] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2023-03-22 08:34:33,295] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-03-22 08:34:33,313] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): sukses_Proses_Load_Datamart> on 2023-03-22 08:34:25.490061+00:00
[2023-03-22 08:34:33,318] {standard_task_runner.py:52} INFO - Started process 794 to run task
[2023-03-22 08:34:33,322] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'load_to_datamart', 'sukses_Proses_Load_Datamart', 'manual__2023-03-22T08:34:25.490061+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/mart_layer.py', '--cfg-path', '/tmp/tmpjop024xk', '--error-file', '/tmp/tmphv4z7cbi']
[2023-03-22 08:34:33,323] {standard_task_runner.py:77} INFO - Job 19: Subtask sukses_Proses_Load_Datamart
[2023-03-22 08:34:33,373] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_datamart.sukses_Proses_Load_Datamart manual__2023-03-22T08:34:25.490061+00:00 [running]> on host 2bbf1c230888
[2023-03-22 08:34:33,435] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_datamart
AIRFLOW_CTX_TASK_ID=sukses_Proses_Load_Datamart
AIRFLOW_CTX_EXECUTION_DATE=2023-03-22T08:34:25.490061+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-22T08:34:25.490061+00:00
[2023-03-22 08:34:33,437] {logging_mixin.py:109} INFO - Etl Sukses Dilakukan
[2023-03-22 08:34:33,437] {python.py:152} INFO - Done. Returned value was: None
[2023-03-22 08:34:33,451] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=load_to_datamart, task_id=sukses_Proses_Load_Datamart, execution_date=20230322T083425, start_date=20230322T083433, end_date=20230322T083433
[2023-03-22 08:34:33,495] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-22 08:34:33,531] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
